### ISSUE TITLE
resizable columns

### ISSUE THREAD

**Issue by turboc1208** [home-assistant/appdaemon#500]
Can the columns in the admin interface be made resizable?  Some of my callback names are larger than the field, but columns like pinned have a lot more space provided than is needed.

**Comment by Odianosen25** [comment]
Well this will be coming surely, but not for now.

You can leave the issue opened so Andrew remembers.

Regards

**Comment by kbuc** [comment]
Yes, please. Or at least add the keyword tooltip in the Execution time column, just like in the Kwargs column.
![Clipboard01](https://user-images.githubusercontent.com/28578761/83809509-d1a91800-a6b6-11ea-9197-d5402b7be3cb.png)
